charts/victoria-metrics-k8s-stack/values.yaml

```
@@ -0,0 +1,451 @@
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+## Create default rules for monitoring the cluster
+##
+defaultRules:
+  create: true
+  rules:
+    etcd: true
+    general: true
+    k8s: true
+    kubeApiserver: true
+    kubeApiserverAvailability: true
+    kubeApiserverSlos: true
+    kubelet: true
+    kubePrometheusGeneral: true
+    kubePrometheusNodeRecording: true
+    kubernetesApps: true
+    kubernetesResources: true
+    kubernetesStorage: true
+    kubernetesSystem: true
+    kubeScheduler: true
+    kubeStateMetrics: true
+    network: true
+    node: true
+
+  ## Runbook url prefix for default rules
+  runbookUrl: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#
+  ## Reduce app namespace alert scope
+  appNamespacesTarget: ".*"
+
+  ## Labels for default rules
+  labels: {}
+  ## Annotations for default rules
+  annotations: {}
+
+  ## Additional labels for PrometheusRule alerts
+  additionalRuleLabels: {}
+
+
+
+##############
+
+# victoria-metrics-operator dependency chart configuration. For possible values refer to https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-operator#parameters
+victoria-metrics-operator:
+  ## all values for victoria-metrics-operator helm chart can be specified here
+  createCRD: false
+  operator:
+    # -- By default, operator converts prometheus-operator objects.
+    disable_promethues_converter: true
+
+
+
+vmsingle:
+  enabled: true
+  # spec for VMSingle crd
+  # https://github.com/VictoriaMetrics/operator/blob/master/docs/api.MD#vmsinglespec
+  spec:
+    retentionPeriod: "14"
+    replicaCount: 1
+
+
+
+alertmanager:
+  enabled: true
+  # spec for VMAlertmanager crd
+  # https://github.com/VictoriaMetrics/operator/blob/master/docs/api.MD#vmalertmanagerspec
+  spec: {}
+
+  # if this one defined, it will be used for alertmanager configuration and config parameter will be ignored
+  # secretName: "alertmanager-config"
+
+  config:
+    global:
+      resolve_timeout: 5m
+    route:
+      group_by: ['job']
+      group_wait: 30s
+      group_interval: 5m
+      repeat_interval: 12h
+      receiver: 'webhook'
+    receivers:
+      - name: 'webhook'
+        webhook_configs:
+          - url: 'http://slack:30500/'
+
+
+
+vmalert:
+  enabled: true
+  # spec for VMAlert crd
+  # https://github.com/VictoriaMetrics/operator/blob/master/docs/api.MD#vmalertspec
+  spec:
+    evaluationInterval: 15s
+    externalLabels:
+      cluster: cluster-name
+    remoteRead:
```

```suggestion
    remoteRead: {}
```

 Lets remove lookback. It will evaluate alerts with 1h delay.